src/Compilers/CSharp/Test/Semantic/Semantics/LambdaTests.cs

```diff
@@ -8553,5 +8553,40 @@ public void ParamsArray_ThisModifier_02()
                 // var lam = (params this int[] xs) => xs.Length;
                 Diagnostic(ErrorCode.ERR_ThisInBadContext, "this").WithLocation(1, 19));
         }
+
+        [Fact]
+        public void CompilerLoweringPreserveAttribute_01()
```

Comment:
Is it worth adding some tests for lambdas observed as expression trees, to make sure there's nothing different about how we do the machinery lowering for that scenario?

Reply:
> Is it worth adding some tests for lambdas observed as expression trees, to make sure there's nothing different about how we do the machinery lowering for that scenario?

I do not think so.